Cargo.toml

```diff
@@ -85,7 +85,7 @@ optional = true
 
 [dependencies.tokio]
 version = "0.2.18"
-features = ["io-util", "sync"]
+features = ["io-util", "sync", "macros"]
```

Comment:
The most objective way to measure would be to compare the binary size of a crate with and without the macros added. I agree that they likely won't make a huge difference, and in the case of users who use tokio, they're likely going to end up including it anyways to be able to use `tokio::main`.

Reply:
The binary size seems to differ by 1K on my machine when building with async-std, so it doesn't seem to be an issue.

Reply:
Yeah, I don't think it's an issue. If users really need a minimal binary, they can strip it, which will likely remove part of that 1K since we aren't using every macro (along with large parts of our other dependencies as well).